.gitlab-ci.yml

```diff
@@ -255,10 +253,10 @@ variables:
 # Condition mixins for simplification of rules
 #
 .if_main_branch: &if_main_branch
-  if: $CI_COMMIT_BRANCH == "main"
+  if: $CI_COMMIT_BRANCH == "main" || $CI_COMMIT_BRANCH == "6.53.x"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:**  Should this be only `6.53.x` and should it be a constant such as `AGENT_MAIN_BRANCH`?‏